.gitignore

```diff
@@ -1,5 +0,0 @@
-typings
-node_modules
-.DS_Store
-.tmp
-dist
```

Comment:
Please restore this file. It prevents us from including build artifacts and other useless files.